website/docs/r/search_index.html.markdown

```diff
@@ -0,0 +1,243 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: search index"
+sidebar_current: "docs-mongodbatlas-resource-search-index"
+description: |-
+Provides a Search Index resource.
+---
+
+# mongodbatlas_search_index
+
+`mongodbatlas_search_index` provides a Search Index resource. This allows indexes to be created.
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_search_index" "test" {
+  name   = "project-name"
+  project_id = "<PROJECT_ID>"
+  cluster_name = "<CLUSTER_NAME>"
+  
+  analyzer = "lucene.standard"
+  collectionName = "collection_test"
+  database = "database_test"
+  mappings_dynamic = true
+  
+  searchAnalyzer = "lucene.standard"
```

Comment:
Definitely need a this example and a more advanced with howing how it would use a custom analyzer (i.e. analyzers) that would have helped me see what you are doing here more clearly and would help our customers.